src/ui/public/agg_response/tabify/_buckets.js

```diff
@@ -68,7 +68,9 @@ TabifyBuckets.prototype._isRangeEqual = function (range1, range2) {
 TabifyBuckets.prototype._orderBucketsAccordingToParams = function (params) {
   if (params.filters && this.objectMode) {
     this._keys = params.filters.map(filter => {
-      return filter.label || filter.input.query || '*';
+      const query = _.get(filter, 'input.query.query_string.query', filter.input.query);
```

Comment:
are we sure `filter.input` is always set ?

Reply:
Yes, we'll throw an error before even sending the request if `input` is undefined https://github.com/elastic/kibana/blob/3ea4c3e0bc9e1557c0515651d9cc5aaf3ea7ad8b/src/ui/public/agg_types/buckets/filters.js#L45